daemon/logger/fluentd/fluentd.go

```diff
@@ -0,0 +1,40 @@
+package fluentd
+
+import (
+	"github.com/docker/docker/daemon/logger"
+	"github.com/fluent/fluent-logger-golang/fluent"
+)
+
+type Fluentd struct {
+	containerid string
+	writer      *fluent.Fluent
+}
+
+func New(tag string) (logger.Logger, error) {
+	log, err := fluent.New(fluent.Config{FluentPort: 24224, FluentHost: "localhost"})
```

Comment:
Most frequent use-case seems to send logs to Fluentd in other container on the same host.

Of course, configurable port and host are better than non-configurable.
Are there any good idea for how to specify host/port? Adding new command line options?